### DISCUSSION TITLE
Scrolling issues on my page

### DISCUSSION THREAD

**Discussion by AdagioB** [quasarframework/quasar#14437] [General - Components / Directives / etc] [answered]
I'm having an issue here that is giving me quite the headache. I have tried a lot of different things, but so far nothing useful

I created a Codepen to showcase a simplified version my site. This is an example without all my tries
https://codepen.io/Adagio_B/pen/VwXjryp?editors=1010

Quick description:
My site has a header and a drawer with menu items in the left side. 
In the content section I have a few buttons in the top, inside a toolbar. Number of buttons vary greatly from page to page. Below the toolbar there are either one or two tables. When you click on a row it will expand with a section containing a lot of input fields (and other type of data). Number and size of input fields can vary a lot. 

What I'm trying to accomplish is the following:
The contents of the left drawer (the menu) and the buttons in the toolbox should not scroll. They should always stay where they are. 
Basically it should only be the area with the two tables that scrolls. 
I have tried to look at the q-scrollarea, but this needs a height to be set in either pixels or vh, but I haven't been able to figure out how to calculate the remaining height of the area. My try is in the example, but disabled by comments. 

My dream scenario would be that the header of the tables would be sticky, so only the contents will scroll. Though from what I can see, this is only possible if I can set the height of the tables, unfortunately I haven't been able to figure out how to calculate this
I used [Tables: Sticky header](https://quasar.dev/vue-components/table#sticky-header-column) to try to get it working, but no success so far

Any suggestions on what I can do?

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/poVPxRE?editors=1000

**Reply by AdagioB** [reply]
Thank you. This seems to be what is needed. 
For some unknown reason it doesn't work in my actual code that is more complicated than my example (too complicated for codepen), but the setup should be the same. I need to dig into it to see if I can find the differences. Thanks again, have a great day

**Comment by pdanpdan** [comment]
- use L instead of l in view in middle group to make drawer fixed
- do not set padding/margin on mount point
- use a styleFn that sets height instead of min-height on QPage
- use column no-wrap on QPage
- use col on the children that should expand/fill the space

**Reply by AdagioB** [reply]
Thank you. I had missed one of the changes, it seems to work now
